docs: add instructions on how to use priority for menu items (#11305)
owncloud/web: docs/extension-system/extension-types/app-menu-items.md

A `url` will result in an `<a>` element that links to an external page. External pages always open in a new tab or window.

At least one of these properties has to be provided when registering an extension. If you define more than one, the priority order is `handler` > `path` > `url`.

> `priority` specifies the order of the menu item. 50 is probably a good place to start, then go up/down based on where the item should be placed. Defaults to the highest possible number, so the item will most likely end up at the bottom of the list.

Reviewer: ```suggestion
`priority` specifies the order of the menu items. 50 is a good number to start with, then go up/down based on where the item should be placed. Defaults to the highest possible number, so the item will most likely end up at the bottom of the list if you don't specify a `priority`.
```

Reviewer: Do we have a recommendation? Not setting anything? Or trying to define a sane priority?

Author: Hard to say, is it valid to not have a recommendation? We explained the effects and how to use it, now it's up to the app creator what to choose I'd say 🤷

Reviewer: Then maybe `Leave it empty if unsure what to pick`?
